cpp/src/parquet/arrow/schema.h

```diff
@@ -104,7 +104,7 @@ struct PARQUET_EXPORT SchemaField {
 /// Expose parquet columns as a tree structure. Useful traverse and link
 /// between arrow's Schema and parquet's Schema.
 struct PARQUET_EXPORT SchemaManifest {
-  static ::arrow::Status Make(
+  static ::arrow::Result<std::unique_ptr<FileReader>> Make(
```

Comment:
Could you work on this as a separated task?

Reply:
Alright, I have reverted this now.